tests/octoberSky/output-expected/0.uorocketlog

```diff
@@ -1,3 +1,4 @@
+Timestamp (Relative),State Number,roll,pitch,yaw,rollAccuracy,pitchAccuracy,yawAccuracy,gpsLatitude,gpsLongitude,gpsAltitude,barometricAltitude,relativeBarometricAltitude,velocityN,velocityE,velocityD,filteredXaccelerometer,filteredYaccelerometer,filteredZaccelerometer,solutionStatus,gpsPosStatus,gpsPosAccurracyLatitude,gpsPosAccuracyLongitude,gpsPosAccuracyAltitude,NumSvUsed,velocityNAccuracy,velocityEAccuracy,velocityDAccuracy,latitudeAccuracy,longitudeAccuracy,altitudeAccuracy,pressureStatus,barometricPressure,imuStatus,gyroX,gyroY,gyroZ,temp,deltaVelX,deltaVelY,deltaVelZ,deltaAngleX,deltaAngleY,deltaAngleZ,loggerInitialized, sbgInitialized,
 52110393763,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,
```

Comment:
```suggestion
```

Deleting this line will fix it. The simulation code previously ignored the first line, even if it wasn't actually the headers. This is why the first two used to be identical. Since we now have real headers, we can delete the extra identical line.